.github/workflows/ci.yml

```diff
@@ -0,0 +1,18 @@
+---
+name: CI for dactyl
+on:
+  pull_request:
+    branches:
+      - master
+jobs:
+  ci:
+    name: Run checks and tests over ${{matrix.otp_vsn}} and ${{matrix.os}}
+    runs-on: ${{matrix.os}}
+    strategy:
+      matrix:
+        otp_vsn: [19.3, 20.3, 21.3, 22.3, 23.1]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v2
+      - name: version
+        run: make version
```

Comment:
We're going to start using GitHub actions instead of Travis CI? Or is this just an experiment?

Reply:
We're going to use it. It's already active in this repo. The pull request replaces one with the other.